internal/postprocessor/config.yaml

```diff
@@ -555,4 +559,111 @@ service-configs:
+   import-path: cloud.google.com/go/containeranalysis/apiv1beta1
+ - input-directory: google/devtools/clouddebugger/v2
+   service-config: clouddebugger_v2.yaml
+   import-path: cloud.google.com/go/debugger/apiv2
+ - input-directory: google/cloud/dialogflow/v2beta1
+   service-config: dialogflow_v2beta1.yaml
+   import-path: cloud.google.com/go/dialogflow/apiv2beta1
+ - input-directory: google/devtools/clouderrorreporting/v1beta1
+   service-config: clouderrorreporting_v1beta1.yaml
+   import-path: cloud.google.com/go/errorreporting/apiv1beta1
+ - input-directory: google/cloud/eventarc/publishing/v1
+   service-config: eventarcpublishing_v1.yaml
+   import-path: cloud.google.com/go/eventarc/publishing/apiv1
+ - input-directory: google/cloud/kms/inventory/v1
+   service-config: kmsinventory_v1.yaml
+   import-path: cloud.google.com/go/kms/inventory/apiv1
+ - input-directory: google/monitoring/v3
+   service-config: monitoring.yaml
+   import-path: cloud.google.com/go/monitoring/apiv3/v2
+ - input-directory: google/cloud/networkconnectivity/v1
+   service-config: networkconnectivity_v1.yaml
+   import-path: cloud.google.com/go/networkconnectivity/apiv1
+ - input-directory: google/cloud/oslogin/v1
+   service-config: oslogin_v1.yaml
+   import-path: cloud.google.com/go/oslogin/apiv1
+ - input-directory: google/cloud/oslogin/v1beta
+   service-config: oslogin_v1beta.yaml
+   import-path: cloud.google.com/go/oslogin/apiv1beta
+ - input-directory: google/cloud/recaptchaenterprise/v1
+   service-config: recaptchaenterprise_v1.yaml
+   import-path: cloud.google.com/go/recaptchaenterprise/v2/apiv1
+ - input-directory: google/cloud/recaptchaenterprise/v1beta1
+   service-config: recaptchaenterprise_v1beta1.yaml
+   import-path: cloud.google.com/go/recaptchaenterprise/v2/apiv1beta1
+ - input-directory: google/cloud/resourcemanager/v2
+   service-config: cloudresourcemanager_v2.yaml
+   import-path: cloud.google.com/go/resourcemanager/apiv2
+ - input-directory: google/api/serviceusage/v1
+   service-config: serviceusage_v1.yaml
+   import-path: cloud.google.com/go/serviceusage/apiv1
+ - input-directory: google/storage/v2
+   service-config: storage_v2.yaml
+   import-path: cloud.google.com/go/storage/internal/apiv2
+ - input-directory: google/devtools/cloudtrace/v1
+   service-config: cloudtrace_v1.yaml
+   import-path: cloud.google.com/go/trace/apiv1
+ - input-directory: google/devtools/cloudtrace/v2
+   service-config: cloudtrace_v2.yaml
+   import-path: cloud.google.com/go/trace/apiv2
+ - input-directory: google/cloud/translate/v3
+   service-config: translate_v3.yaml
+   import-path: cloud.google.com/go/translate/apiv3
+ - input-directory: google/cloud/vision/v1
+   service-config: vision_v1.yaml
+   import-path: cloud.google.com/go/vision/v2/apiv1
+ - input-directory: google/cloud/vision/v1p1beta1
+   service-config: vision_v1p1beta1.yaml
+   import-path: cloud.google.com/go/vision/v2/apiv1p1beta1
+ - input-directory: google/cloud/workstations/v1beta
```

Comment:
Unsorted, move `workstations` down?

Reply:
It is sorted, maybe this was in an older commit